.circleci/config.yml

```diff
@@ -145,9 +146,68 @@ jobs:
   end_to_end_tests:
     <<: *defaults
     environment:
-      SPEC_FILE_PATTERN: ./specs/core/**/*.spec.js
+      SPEC_FILE_PATTERN: ./remote/specs/core/**/*.spec.js
     <<: *run_e2e_and_save_artifacts
 
+  end_to_end_tests_localdb:
+    working_directory: /tmp/repo
+    machine:
+      enabled: true
+      image: ubuntu-1604:201903-01
```

Comment:
Not very important to address this but just curious: is this the recommended image to use when doing docker stuff on circleci? Seems a bit excessive to use a full ubuntu install?

Reply:
No, using a machine here is crucial since this is the simplest way to able different docker containers to communicate with each other (as far as we could see).